testproject/Assets/Tests/Runtime/Support/SpawnRpcDespawnInstanceHandler.cs

```diff
@@ -55,15 +55,15 @@ public NetworkObject Instantiate(ulong ownerClientId, Vector3 position, Quaterni
             return networkObject;
         }
 
-        public void Destroy(NetworkObject networkObject)
+        public bool Destroy(NetworkObject networkObject)
```

Comment:
Unclear to me what true / false means - function comment block pls?

Reply:
I could add a comment in the unit test, but it is explained in the [INetworkPrefabInstanceHandler](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/blob/831c3a3b2480a3391b4acff76b39d5805c46cc6a/com.unity.netcode.gameobjects/Runtime/Spawning/NetworkPrefabHandler.cs#L44) where if you return true then the GameObject with the NetworkObject component in question will be destroyed, otherwise (false) it will simply be removed from the spawned lists.  
If you still would like to see a comment in this unit test let me know? ( this was me just making adjustments to any INetworkPrefabInstanceHandler implementation ).  There is [a more detailed explanation above](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/955#discussion_r690249622) as to why this adjustment was made.